website/docs/releases/2024/v2024.8.md

```diff
@@ -66,22 +73,36 @@ To try out the release candidate, replace your Docker image tag with the latest
 
 -   **Source property mappings for SCIM, OAuth, SAML and Plex sources**
 
+    All source types now support Property mappings to customize how authentik should interpret the data the source provides. In addition to that, it is also now possible to sync groups and group membership from sources that provide group information. See [Property Mappings](../../sources/property-mappings/index.md).
+
 -   **RADIUS provider custom attribute support**
 
     With 2024.8 it is possible to define custom attributes for the RADIUS provider, for example vendor specific attributes like Cisco's `AV-Pair` attribute. These attributes are defined in property mappings which means they can be dynamically defined based on the user authenticating. See [RADIUS Provider](../../providers/radius/index.mdx#radius-attributes)
 
--   **SAML Source encryption support**
+-   **SAML encryption support**
+
+    It is now possible to configure a SAML Source/Provider to decrypt and validate encrypted assertions. This can be configured by certain a [Certificate-keypair](../../core/certificates.md) and selecting it in the SAML Source/Provider.
```

Comment:
```suggestion
    It is now possible to configure a SAML source and/or provider to decrypt and validate encrypted assertions. This can be configured by creating a [Certificate-keypair](../../core/certificates.md) and selecting it in the SAML source os provider.
```

Reply:
```suggestion
It is now possible to configure a SAML sources and providers to decrypt and validate encrypted assertions. This can be configured by creating a [Certificate-keypair](../../core/certificates.md) and selecting it in the SAML source or provider.
```